rmgpy/molecule/molecule.pxd

```diff
@@ -61,7 +61,15 @@ cdef class Atom(Vertex):
 
     cpdef bint isOxygen(self)
 
+    cpdef bint isFluorine(self)
+
+    cpdef bint isSilicon(self)
```

Comment:
Make a note in the commit that `isSilicon` was added to cython file as well, good catch!